src/io/github/sspanak/tt9/ime/helpers/InputType.java

```diff
@@ -51,7 +51,7 @@ public boolean isDialer() {
 		int inputType = field.inputType & android.text.InputType.TYPE_MASK_CLASS;
 
 		return
-			inputType == android.text.InputType.TYPE_CLASS_PHONE && field.packageName.equals("com.android.dialer");
+			inputType == android.text.InputType.TYPE_CLASS_PHONE && field.packageName.equals("com.android.dialer") || inputType == android.text.InputType.TYPE_CLASS_NUMBER;
```

Comment:
This is an overkill for just fixing the calculator. It will hide the UI in any numeric field, but this is unnecessary most of the time and more important, it will make the on-screen keyboard unusable when one actually needs to type numbers.

Isn't it possible to use the `packageName` and match it to your calculator app instead?